vpr/src/base/read_options.cpp

```diff
@@ -2043,9 +2073,15 @@ void set_conditional_defaults(t_options& args) {
     }
 
     //Are we using the automatic, or user-specified annealing schedule?
-    if (args.PlaceInitT.provenance() == Provenance::SPECIFIED
-        || args.PlaceExitT.provenance() == Provenance::SPECIFIED
-        || args.PlaceAlphaT.provenance() == Provenance::SPECIFIED) {
+    if (args.PlaceAlphaMin.provenance() == Provenance::SPECIFIED
+        || args.PlaceAlphaMax.provenance() == Provenance::SPECIFIED
+        || args.PlaceAlphaDecay.provenance() == Provenance::SPECIFIED
+        || args.PlaceSuccessMin.provenance() == Provenance::SPECIFIED
+        || args.PlaceSuccessTarget.provenance() == Provenance::SPECIFIED) {
+        args.anneal_sched_type.set(DUSTY_SCHED, Provenance::INFERRED);
```

Comment:
I assume this structure allows for inferred and explicit selection of the new schedule?

Reply:
I've added comments.